design-library/src/components/BccTabs/BccTabs.css

```diff
@@ -7,6 +7,22 @@
         @apply relative flex items-end min-w-full overflow-scroll hide-scrollbar;
     }
 
+    .bcc-tabs--fill {
```

Comment:
Nitpick: elsewhere I've used a single `-` for modifiers as well - deliberately to keep it simple and not end up in a full BEM syntax. For consistency I would suggest to do the same here, `bcc-tabs-fill`.